src/test/scala/uk/gov/hmrc/perftests/itsass/EmploymentsRequests.scala

```diff
@@ -24,24 +24,109 @@ import uk.gov.hmrc.perftests.itsass.RequestsHelper._
-    .get(s"$employmentsUrl/employment-summary")
+  def getEmploymentSummaryPage(taxYear: String): HttpRequestBuilder = http("Get Employment Summary Page")
+    .get(s"${employmentsUrl(taxYear)}/employment-summary")
     .check(status.is(200))
 
-  def getEmployerDetailsAndBenefitsPage(employmentId: String): HttpRequestBuilder = http("Get Employer Details And Benefits Page")
-    .get(s"$employmentsUrl/employer-details-and-benefits?employmentId=$employmentId")
-    .check(saveCsrfToken())
+  def getEOYEmploymentSummaryPage(taxYearEOY: String): HttpRequestBuilder = http("Get Employment Summary Page")
+    .get(s"${employmentsUrl(taxYearEOY)}/employment-summary")
+    .check(saveCsrfToken)
+    .check(status.is(200))
+
+  def postEmploymentSummaryPage(taxYear: String): HttpRequestBuilder = http("Post Employment Summary Page")
+    .post(s"${employmentsUrl(taxYear)}/employment-summary")
+    .formParam("""csrfToken""", """${csrfToken}""")
+    .formParam("value", true)
+    .check(saveEmploymentId)
+    .check(status.is(303))
+
+  def getEmployerDetailsAndBenefitsPage(taxYear: String, employmentId: String): HttpRequestBuilder = http("Get Employer Details And Benefits Page")
+    .get(s"${employmentsUrl(taxYear)}/employer-details-and-benefits?employmentId=$employmentId")
+    .check(saveCsrfToken)
+    .check(status.is(200))
+
+  def getEmploymentDetailsPage(taxYear: String, employmentId: String): HttpRequestBuilder = http("Get Employment Details Page")
+    .get(s"${employmentsUrl(taxYear)}/check-employment-details?employmentId=$employmentId")
     .check(status.is(200))
 
-  def getEmploymentDetailsPage(employmentId: String): HttpRequestBuilder = http("Get Employment Details Page")
-    .get(s"$employmentsUrl/check-employment-details?employmentId=$employmentId")
+  def getEmploymentBenefitsPage(taxYear: String, employmentId: String): HttpRequestBuilder = http("Get Employment Benefits Page")
+    .get(s"${employmentsUrl(taxYear)}/check-employment-benefits?employmentId=$employmentId")
     .check(status.is(200))
 
-  def getEmploymentBenefitsPage(employmentId: String): HttpRequestBuilder = http("Get Employment Benefits Page")
-    .get(s"$employmentsUrl/check-employment-benefits?employmentId=$employmentId")
+  def getEmploymentExpensesPage(taxYear: String): HttpRequestBuilder = http("Get Employment Expenses Page")
+    .get(s"${employmentsUrl(taxYear)}/check-employment-expenses")
     .check(status.is(200))
 
-  def getEmploymentExpensesPage: HttpRequestBuilder = http("Get Employment Expenses Page")
-    .get(s"$employmentsUrl/check-employment-expenses")
+  def getAddEmploymentPage(taxYear: String): HttpRequestBuilder = http("Get Add Employment Page")
+    .get(s"${employmentsUrl(taxYear)}/add-employment")
+    .check(saveCsrfToken())
     .check(status.is(200))
+
+  def postAddEmploymentPage(taxYear: String): HttpRequestBuilder = http("Post Add Employment Status Page")
```

Comment:
Post Add Employment Status Page, do you need the "status" in the name of the page?

Reply:
done